### PR TITLE
Adjusts the emergency shuttle call times and makes the shuttle autocall if the round goes past 2 hours

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

AUTOEVAC: 15 minutes (was 25)
GREEN: 12 minutes (was 20)

makes shuttle autocall if the round goes past 2 hours, because everything should have been finished by then.

# Why is this good for the game?

- less time waiting for a shuttle for rounds "ended" (hopelessly lost)
- more pressure on sec to finish the round up by getting antagonists (less camping)
- more pressure on antagonists to do things (you cant just do all your objectives in the 20 minute call)

# Changelog

:cl:  
tweak: adjusts the values of autoevac and green alert emergency shuttle times (25 -> 15, 20 -> 12)
rscadd: the shuttle autocalls if the round goes past 2 hours
/:cl:

### CODE REVIEW

**Comment by iamthesilly** [comment]
Should probably also autocall if all crew are dead

**Comment by ShoziOffical** [comment]
> Should probably also autocall if all crew are dead

It's bugged

**Comment by ShoziOffical** [comment]
> Should probably also autocall if all crew are dead

It's bugged

**Comment by ktlwjec0** [comment]
I assume you know but just in case - the alert still comes up for lack of response even if the shuttle is already in transit.

**Review by Moltijoe — CHANGES_REQUESTED** [review]
make sure it doesn't trigger if it's already been called

File: code/controllers/subsystem/shuttle.dm [hunk 1]
```diff
@@ -217,28 +217,36 @@ SUBSYSTEM_DEF(shuttle)
...
+
+		if(alive / total <= threshold)
+			var/msg = "Automatically dispatching shuttle due to crew death."
+			message_admins(msg)
+			log_game("[msg] Alive: [alive], Roundstart: [total], Threshold: [threshold]")
+			emergency_no_recall = TRUE
+			priority_announce("Catastrophic casualties detected: crisis shuttle protocols activated - jamming recall signals across all frequencies.")
+			if(emergency.timeLeft(1) > emergency_no_recall * 0.4)
+				emergency.request(null, set_coefficient = 0.4)
+			return
```
Comment (Moltijoe) [inline code/controllers/subsystem/shuttle.dm]: add a check for if the shuttle is already called here too

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**